code/modules/vehicles/hoverdog.dm

```diff
@@ -0,0 +1,113 @@
+
+//jeep, basically
+
+/obj/vehicle/ridden/hoverdog
+	name = "XXL-9a 'Hoverdog'"
+	desc = "A top of the line hoverbike, comissioned by Nanotrassen for their hotdog salespeople."
+	icon = 'icons/obj/car.dmi'
+	icon_state = "hoverdog"
+	layer = LYING_MOB_LAYER
+	pixel_y = -48
+	pixel_x = -48
+	max_buckled_mobs = 1
+	max_occupants = 1
+	pass_flags_self = null
+	max_integrity = 200
+	cover_amount = 45
+	armor_type = /datum/armor/weeweenor
+	var/crash_all = FALSE
+	move_force = MOVE_FORCE_VERY_STRONG
+	move_resist = MOVE_FORCE_EXTREMELY_STRONG
+	key_type = /obj/item/key/hoverdog
+	integrity_failure = 0.2
+	var/crash_dmg_high = 0
+	var/crash_dmg_low = 0
+	var/crash_dmg_stm = 50
+	var/crash_para_driv = 1.2
+	var/crash_para_pass = 0.3
+	var/crash_para_roadkill = 0.9
+
+/datum/armor/weeweenor
+	melee = 10
+	bullet = 10
+	laser = 10
+	energy = 10
+	fire = 30
+	acid = 30
+
+
+/obj/vehicle/ridden/hoverdog/Initialize(mapload)
+	. = ..()
+	add_overlay(image(icon, "hoverdog_cover", ABOVE_MOB_LAYER))
+	AddElement(/datum/element/ridable, /datum/component/riding/vehicle/hoverdog)
+
+/obj/vehicle/ridden/hoverdog/Bump(mob/living/carbon/human/rammed)
+	. = ..()
+	if(!ishuman(rammed) || !rammed.density || !has_buckled_mobs())
+		return
+	rammed.stamina?.adjust(-crash_dmg_stm)
+	rammed.apply_damage(rand(crash_dmg_low, crash_dmg_high), BRUTE)
+	for(var/mob/living/rider in buckled_mobs)
+		var/paralyze_time = is_driver(rider) ? crash_para_driv : crash_para_pass
+		rider.Paralyze(paralyze_time SECONDS)
+	rammed.Paralyze(crash_para_roadkill SECONDS)
```

Comment:
oh god do you really want this thing to actually stun whoever it rams into

also please add log messages for when it rams into someone, something like this maybe:
```dm
log_combat(src, rammed, "rammed into")
```